MAUI/TabView/Swiping.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: post
+title: About .NET MAUI Tab View (SfTabView) control | Syncfusion
+description: Learn here all about swiping support in Syncfusion .NET MAUI Tab View (SfTabView) control and more.
+platform: maui
+control: Tab View
+documentation: ug
+---
+
+# Swiping in .NET MAUI Tab View (SfTabView)
+
+The `EnableSwiping` property of `SfTabView` is used to navigate between tab contents using swiping. The default value of `EnableSwiping` is `true.`
```

Comment:
The `EnableSwiping` property of `SfTabView` allows users to switch between tab contents by swiping. By default, the `EnableSwiping` property is set to `true`.

Reply:
Yes, I changed the content.